client/components/user-home.js

```diff
@@ -1,55 +1,98 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import {connect} from 'react-redux'
-import {PlayerAdd, PlayerList, GameCard, GameActionButton} from '../components'
+import {
+  PlayerAdd,
+  PlayerList,
+  EventCard,
+  EventActionButton
+} from '../components'
+import {fetchAllEvents} from '../store/event'
 
 /**
  * COMPONENT
  */
-export const UserHome = props => {
-  const {email} = props
+export class UserHome extends React.Component {
+  constructor(props) {
+    super(props)
+  }
 
-  return (
-    <div className="container">
-      <h5>Welcome, {email}</h5>
-      <div className="divider" />
-      <div className="section">
-        {' '}
-        <h6>Pending Games</h6>{' '}
-      </div>
-      <div className="divider" />
-      <div className="row">
-        {[1, 2, 3, 4, 5, 6, 7, 8].map(card => (
-          <GameCard key={card} type="pending" />
-        ))}
-      </div>
-      <div className="section">
-        {' '}
-        <h6>Completed Games</h6>{' '}
-      </div>
-      <div className="divider" />
-      <div className="row">
-        {[1, 2, 3, 4, 5, 6, 7, 8].map(card => (
-          <GameCard key={card} type="done" />
-        ))}
-      </div>
-      {/* <PlayerAdd />
+  componentDidMount() {
+    this.props.getUsersEvents()
```

Comment:
This should be init'd at a higher level so we know it will always be there?

Reply:
Actually, you DID init it at a higher level.  So this is extra, and can be deleted (this whole component can be functional).